projects/peerdas-metrics.md

```diff
@@ -0,0 +1,78 @@
+# PeerDAS Metrics
+
+Improving PeerDAS testing efficiency via metrics implementation and visualization. 
+
+## Motivation
+
+PeerDAS is now entering its testing phase, with most CL clients actively participating. Identifying and utilizing specific metrics is crucial for evaluating efficiency, troubleshooting, and optimizing the protocol. Before PeerDAS is deployed on the mainnet, detailed testing is important to keep the Ethereum Protocol secure and efficient. The project aims to analyze and present these results in an accessible format for core developers and the research team.
+
+## Project description
+
+PeerDAS Metrics focuses on identifying the most crucial metrics, integrating them into CL clients (Teku, Grandine), and developing visual representations to help teams use these metrics effectively. Additionally, the project will present the analysis of these metrics, providing valuable insights for the research team.
```

Comment:
Does this mean creating a new metrics category in clients? I see teku collects 'network', 'libp2p' categories for example, I guess there might be an overlap but I am not sure which one is for blobs. I wonder if you are creating 'peerdas' metrics namespace or just improving existing

Reply:
Every client implements metrics in its own way. Some of these metrics are being added into existing categories. Others were put into new categories. So, it's a bit tricky to categorize them in the similar manner. That's what I am going to find out. The main goal is to get the results from these metrics and to analyze them. I consider it to be helpful for monitoring PeerDAS issues.